src/Runner.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Pantheon\Terminus;
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+class Runner
+{
+    /**
+     * @var Terminus
+     */
+    private $application;
+    /**
+     * @var string
+     */
+    private $commands_directory = __DIR__ . '/Commands';
+    /**
+     * @var string
+     */
+    private $top_namespace = 'Pantheon\Terminus\Commands';
+
+    /**
+     * Runner constructor
+     *
+     * @param array $options Options to configure the runner
+     */
+    public function __construct(array $options = [])
+    {
+        $this->application = new Terminus();
```

Comment:
I think it'd be better to inject the application object into the runner rather than new it up inside the constructor